doc/api/globals.md

```diff
@@ -637,6 +637,40 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency} logical processors`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
+
+The value is representing the language version as defined in RFC <5646>.
```

Comment:
The `RFC <5646>` should really be a link, e.g. `[RFC 5646][]` with the actual link in footer with the other links.

Reply:
Will check.

Reply:
PTAL